src/components/vm/vmCloneDialog.jsx

```diff
@@ -76,7 +76,7 @@ export const CloneDialog = ({ name, connectionName, toggleModal }) => {
                    </Button>
                </>
            }>
-            <Form isHorizontal>
+            <Form onSubmit={e => e.preventDefault()} isHorizontal>
```

Comment:
Only one tiny comment. if the modal has a single value, then the primary button operation should run on submit. https://github.com/patternfly/patternfly-react/issues/5680#issuecomment-831538214